### PR TITLE
feat: add a note about default limits []

### PR DESCRIPTION
**Pull request by Rowadz**
## Purpose

This might help people to see that the default limit of the `collections` is `100`

https://user-images.githubusercontent.com/38977667/232449691-3c203ec2-8ecb-41cd-b3bd-4ca40f8fdf1f.mp4

### CODE REVIEW

File: apps/graphql-playground/src/components/GqlPlayground.tsx [hunk 1]
```diff
@@ -83,17 +88,41 @@ function GqlPlayground(props: GqlPlaygroundProps) {
         },
       ];
 
-  const settings = { "editor.theme": "light" };
+  const settings = { 'editor.theme': 'light' };
+
+  useEffect(() => {
+    const unsubscribe = store.subscribe(() => {
+      const query: string = getQuery(store.getState());
+      const result = query.toLowerCase().includes('collection');
```
Comment (Rowadz) [inline apps/graphql-playground/src/components/GqlPlayground.tsx:96]: Unfortunately, that's the only way I managed to pull the data from [graphql-playground-react](https://github.com/graphql/graphql-playground/tree/main/packages/graphql-playground-react)

They don't expose the codemirror instance not provide another way to get the query value.

File: apps/graphql-playground/src/components/GqlPlayground.tsx [hunk 2]
```diff
@@ -83,17 +88,41 @@ function GqlPlayground(props: GqlPlaygroundProps) {
...
+      if (hasCollection !== result) {
+        setHasCollection(result);
+      }
```
Comment (Rowadz) [inline apps/graphql-playground/src/components/GqlPlayground.tsx:99]: doing this 
```ts
setHasCollection(query.toLowerCase().includes('collection'));
```

will cause an infinite re-render.

File: apps/graphql-playground/src/components/GqlPlayground.tsx [hunk 3]
```diff
@@ -87,10 +90,36 @@ function GqlPlayground(props: GqlPlaygroundProps) {
...
-    </Provider>
+    <>
+      {hasCollection && (
+        <Note>
+          Did you know that the default limit for a collection is <b>100</b>, lowering this would
+          decrease your query complexity! [
+          <a
+            target="_blank"
+            rel="noreferrer"
+            href="https://www.contentful.com/developers/docs/references/graphql/#/reference/collection-fields/arguments:~:text=default%20is%200-,limit,to%20fetch.%20The%20default%20is%20100%20and%20the%20maximum%20is%201000,-where">
```
Comment (andipaetzold) [inline apps/graphql-playground/src/components/GqlPlayground.tsx:113]: That link should be shortened to `https://www.contentful.com/developers/docs/references/graphql/#/reference/collection-fields/arguments`

File: apps/graphql-playground/src/components/GqlPlayground.tsx (hunk 2 above)
Comment (andipaetzold) [reply apps/graphql-playground/src/components/GqlPlayground.tsx:99]: Even if you remove `hasCollection` from the dependencies?

File: apps/graphql-playground/src/components/GqlPlayground.tsx [hunk 4]
```diff
@@ -87,10 +90,36 @@ function GqlPlayground(props: GqlPlaygroundProps) {
...
+  useEffect(() => {
+    const unsubscribe = store.subscribe(() => {
+      const query: string = getQuery(store.getState());
+      const result = query.toLowerCase().includes('collection');
+      if (hasCollection !== result) {
+        setHasCollection(result);
+      }
+    });
+    return unsubscribe;
+  }, [hasCollection]);
```
Comment (andipaetzold) [inline apps/graphql-playground/src/components/GqlPlayground.tsx:102]: `hasCollection` should be removed from the dependencies here. We currently unsubscribe and resubscribe whenever `hasCollection` changes. We actually only want to subscribe on mount and unsubscribe on unmount.

File: apps/graphql-playground/src/components/GqlPlayground.tsx [hunk 5]
```diff
@@ -87,10 +90,36 @@ function GqlPlayground(props: GqlPlaygroundProps) {
...
+    <>
+      {hasCollection && (
+        <Note>
+          Did you know that the default limit for a collection is <b>100</b>, lowering this would
+          decrease your query complexity! [
+          <a
+            target="_blank"
+            rel="noreferrer"
+            href="https://www.contentful.com/developers/docs/references/graphql/#/reference/collection-fields/arguments:~:text=default%20is%200-,limit,to%20fetch.%20The%20default%20is%20100%20and%20the%20maximum%20is%201000,-where">
+            source
```
Comment (andipaetzold) [inline apps/graphql-playground/src/components/GqlPlayground.tsx:114]: "source" feels quite technical. Maybe "Learn more" without the square brackets?

**Review by andipaetzold — CHANGES_REQUESTED** [review]

File: apps/graphql-playground/src/components/GqlPlayground.tsx (hunk 2 above)
Comment (Rowadz) [reply apps/graphql-playground/src/components/GqlPlayground.tsx:99]: actually I can just do this 

```ts
  useEffect(() => {
    const unsubscribe = store.subscribe(() => {
      const query: string = getQuery(store.getState());
      setHasCollection(query.toLowerCase().includes('collection'));
    });
    return unsubscribe;
  }, []);
```

maybe I did something else that caused the infinite re-render.

Comment (andipaetzold) [reply apps/graphql-playground/src/components/GqlPlayground.tsx]: Yes, that should work

File: apps/graphql-playground/src/components/GqlPlayground.tsx [hunk 6]
```diff
@@ -93,27 +93,23 @@ function GqlPlayground(props: GqlPlaygroundProps) {
...
           Did you know that the default limit for a collection is <b>100</b>, lowering this would
-          decrease your query complexity! [
+          decrease your query complexity!{' '}
           <a
             target="_blank"
             rel="noreferrer"
-            href="https://www.contentful.com/developers/docs/references/graphql/#/reference/collection-fields/arguments:~:text=default%20is%200-,limit,to%20fetch.%20The%20default%20is%20100%20and%20the%20maximum%20is%201000,-where">
-            source
+            href="https://www.contentful.com/developers/docs/references/graphql/#/reference/collection-fields/arguments">
+            learn more
```
Comment (andipaetzold) [inline apps/graphql-playground/src/components/GqlPlayground.tsx:111]: Maybe capitalize "Learn more". Not sure whether lowercase looks good here.

**Review by andipaetzold — APPROVED** [review]

**Pull request merged**